packages/cli-plugin-metro/src/commands/bundle/buildBundle.ts

```diff
@@ -95,14 +70,16 @@ export async function buildBundleWithConfig(
     sourceMapUrl = path.basename(sourceMapUrl);
   }
 
-  const requestOpts: RequestOptions = {
+  const requestOpts: Partial<BundleOptions> = {
     entryFile: args.entryFile,
     sourceMapUrl,
     dev: args.dev,
     minify: args.minify !== undefined ? args.minify : !args.dev,
     platform: args.platform,
-    unstable_transformProfile: args.unstableTransformProfile,
-    generateStaticViewConfigs: args.generateStaticViewConfigs,
+    unstable_transformProfile:
+      (args.unstableTransformProfile as BundleOptions['unstable_transformProfile']) ??
+      'default',
+    // generateStaticViewConfigs: args.generateStaticViewConfigs,
```

Comment:
What's the reason this is commented out?

Reply:
Great spot! I'd actually done this bit wrong — `RequestOptions` is the CLI's input options, and is a superset of Metro's `BundlerOptions`. Updating.